manifests/i/Intel/OneAPI/HPCToolkit/2025.1.3.7/Intel.OneAPI.HPCToolkit.locale.en-US.yaml

```diff
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.4.6 $debug=NVS2.CRLF.7-5-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: Intel.OneAPI.HPCToolkit
+PackageVersion: 2025.1.3.7
+PackageLocale: en-US
+Publisher: Intel Corporation
+# PublisherUrl: https://www.intel.com/content/www/us/en/homepage.html
+PublisherSupportUrl: https://community.intel.com/
+# PrivacyUrl: https://www.intel.com/content/www/us/en/privacy/intel-privacy-notice.html
+# Author:
+PackageName: Intel® oneAPI HPC Toolkit
+# PackageUrl: https://www.intel.com/content/www/us/en/developer/tools/oneapi/hpc-toolkit-download.html
+License: Proprietary
+# LicenseUrl:
+Copyright: © Intel Corporation
+# CopyrightUrl: https://www.intel.com/content/www/us/en/legal/terms-of-use.html
+ShortDescription: Optimized Tools for High-Performance Computing
+Description: |-
+  High-performance computing (HPC) is at the core of AI, machine learning, and
+  deep learning applications. Intel® oneAPI HPC Toolkit delivers what developers
+  need to build, analyze, optimize, and scale HPC applications with the latest
+  techniques in vectorization, multithreading, multi-node parallelization, and
+  memory optimization.
+
+  This toolkit includes powerful data-centric libraries and advanced analysis
+  tools.
+# Moniker:
+# Tags:
+# ReleaseNotes:
+# ReleaseNotesUrl: https://www.intel.com/content/www/us/en/developer/articles/release-notes/oneapi-hpc-toolkit/2025.html
+# PurchaseUrl:
+# InstallationNotes:
```

Comment:
```suggestion
InstallationNotes: |-
  The default installation pass following switches to the installer:

  `-a --silent --eula accept -p=NEED_VS2019_INTEGRATION=0 -p=NEED_VS2022_INTEGRATION=0`

  It WILL NOT integrate this toolkit to VSIDE 2019/2022. If you need this, please
  reinstall this package with `--override` switch and override `NEED_VSXXXX_INTEGRATION`
  as value 1.
```